gatsby-config.js

```diff
@@ -161,6 +161,15 @@ module.exports = {
             }),
           },
         },
+        signup: {
+          environment: process.env.ENVIRONMENT || 'staging',
```

Comment:
we need to add this env var to gatsby cloud. should it be `production`?

Reply:
looks like it should be `production`. but please double check me
![2022-06-29_12-10-07](https://user-images.githubusercontent.com/2952843/176516965-16d5b407-a9ad-4dd3-ae37-0dba83bf58d2.png)

Reply:
yes it should be production, but don't we already have this set in the site??

Reply:
i just added another variable

Reply:
ideally we'd operate from one env var to indicate waht env we're in.

not 100% what would happen if window._nr_signup.environment is set to `development` though (which is the only other value we use for `GATSBY_NEWRELIC_ENV`.

in any case, we can proceed and dig into that further as a potential refactor if needed